packages/errors/src/error-codes.ts

```diff
@@ -103,4 +103,7 @@ export enum ErrorCode {
 
   // graphql
   STREAM_PARSING_ERROR = 'stream-parsing-error',
+
+  // Unknown
+  UNKNOWN_ERROR = 'unknown-error',
```

Comment:
It seems redundant to suffix the codes with `_ERROR.`

Reply:
I see your point, I was following the syntax of the above error, but I can see that most error codes don't have it, should we remove it from the other error codes in a follow up issue?

Reply:
I'd say yes, but that can be left out of this PR.

The `UNKNOWN` is simple and works fine, others may require pondering/renaming.

Reply:
[ffa6470](https://github.com/FuelLabs/fuels-ts/pull/2562/commits/ffa64701a31bde6686bcfc5cf61894457508b4f2)